lib/routes/njupco/index.ts

```diff
@@ -0,0 +1,16 @@
+import { Route } from '@/types';
+import { handler } from './handler-func';
+
+export const route: Route = {
+    path: '/news',
+    categories: ['university'],
+    example: '/njupco/news',
+    radar: [
+        {
+            source: ['http://www.njupco.com/'],
+        }
+    ],
+    name: '南京大学出版社',
```

Comment:
Route name should not be the same as namespace name. https://github.com/DIYgod/RSSHub/blob/755270ccdde6db8ce78c117fe45e9c383d960fbf/lib/routes/njupco/namespace.ts#L4